docs/Modders/uniques.md

```diff
@@ -1890,4 +1887,4 @@ Simple unique parameters are explained by mouseover. Complex parameters are expl
 *[stats]: For example: `+2 Production, +3 Food`. Note that the stat names need to be capitalized!
 *[tech]: The name of any tech.
 *[tileFilter]: Anything that can be used either in an improvementFilter or in a terrainFilter can be used here, plus 'unimproved'
-*[victoryType]: The name of any victory type: 'Neutral', 'Cultural', 'Diplomatic', 'Domination', 'Scientific', 'Time'
+*[victoryType]: The name of any victory type: 'Neutral', 'Cultural', 'Diplomatic', 'Domination', 'Scientific', 'Time'
```

Comment:
Not sure, what has changed here, but probably, this file (same as `DiplomacyManager.kt`) is altered and added by mistake.

Reply:
This file keeps getting updated automatically. I'm not familiar with this process, so I'm not sure why. I'll be sure not to push it in the future.